src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -2349,7 +2350,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
 
     // Method Description:
     // - Get the size of a single character of this control. The size is in
-    //   DIPs. If you need it in _pixels_, you'll need to multiply by the
+    //   _pixels_. If you want it in DIPs, you'll need to DIVIDE by the

```

Comment:
wait, was the contract here just _wrong_?

Reply:
YUP